src/crypto/tls/common.go

```diff
@@ -824,6 +862,10 @@ type Config struct {
 	// See https://tools.ietf.org/html/draft-ietf-tls-subcerts.
 	SupportDelegatedCredential bool
 
+	// AllowKEMTLS is true if the client or server is willing
+	// to start a KEMTLS handshake based on TLS 1.3.
+	AllowKEMTLS bool
```

Comment:
```suggestion
	KEMTLSEnabled bool
```
This aligns better with other flags in `Config` (e.g., `ECHEnabled`).

Reply:
Sure.